packages/types/src/middleware.ts

```diff
@@ -283,6 +283,12 @@ export interface HandlerOptions {
    */
   name?: string;
 
+  /**
+   * @internal
+   * Aliases for the middleware
```

Comment:
expand the explanation: 

aliases allow a middleware to be found by multiple names, used for example in auth refactor to allow older references to the existing middleware names to continue working.